prac_05/wimbledon.py

```diff
@@ -1,3 +1,47 @@
 """
-Time  estimate:
-"""
\ No newline at end of file
+Time  estimate: 40 minutes
+actual: 60 minutes
+"""
+
+
+def main():
+    filename = "wimbledon.csv"
+    champions_data = make_champion_list(filename)
+    name_to_count, champion_countries = count_name_country(champions_data)
+    print("Wimbledon Champions:")
+    for name, count in name_to_count.items():
+        print(f"{name} {count}")
+    print()
+    print(f"These {len(champion_countries)} countries have won Wimbledon:")
+    countries = ", ".join(champion_countries)
```

Comment:
countries is plural which implies a list. However I don't have a better alternative to the variable name.